### PR TITLE
DOC: Add documentation of Data Broker and basic Muxer usage

### PR DESCRIPTION
**Pull request by danielballan**
I took the liberty of pushing this already so that Claudio and Dario can start using it. The new stuff is [here](http://nsls-ii.github.io/broker/basics.html).

This documents the current master of databroker, which as of this writing is not installed anywhere. I assume XPD and CSX will be brought up to date tomorrow and SRX sometime soon.

### CODE REVIEW

**Comment by ericdill** [comment]
This is awesome. :+1: from claudio/dario/stuart/vivek. It is also installed on CSX

File: source/broker/basics.rst [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+excruciating detail.)
+
+Consider this an intermediate step. The data is structured in a generic way
+that is wonderfully fleixble but not especially convenient. To get a more
+useful view of the data, read on.
+
+Putting the Data into a More Useful Form
+----------------------------------------
+
+One level above the DataBroker sits the DataMuxer, an object for synthesizing
```
Comment (tacaswell) [inline source/broker/basics.rst]: synthesizing -> 'merging and aligning' .  Synthesizing sounds like fake data.

File: source/broker/basics.rst [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+streams of Events from mamy sources into a table. Build a DataMuxer like so:
+
+.. ipython:: python
+
+   from dataportal.muxer import DataMuxer
+   dm = DataMuxer.from_events(events)
+
+The ``events`` can be from one scan or from many scans together. Then, the
+implest task is to simply look at the data from one source -- say, sample
```
Comment (tacaswell) [inline source/broker/basics.rst]: sp

File: source/broker/basics.rst [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+
+Next, let's obtain a table showing data from multiple sources. Strictly
+speaking, measurements recorded by different equipment are not in general
+synchronized, but in practice one usually ignores small differences in time.
+For instance, we might want to plot "temperature" versus "intensity" even if
+the temperature and intesity sensors never happened to take a simultaneous
+measurement. Doing so, we would be implicitly *binning* those measurements
+in time.
+
+Therfore, plotting one dependent variable against another usually requires
```
Comment (tacaswell) [inline source/broker/basics.rst]: sp

**Comment by danielballan** [comment]
Thanks, @ericdill.

**Comment by ericdill** [comment]
This looks like it is ready to go. Any reason not to merge this?

**Comment by danielballan** [comment]
I just force-pushed a last round of copy edits. Ready to go.

**Comment by ericdill** [comment]
Awesome.

**Pull request merged**